api/color_scheme.inc

```diff
@@ -110,21 +109,19 @@ function tripald3_get_scheme_colors($scheme_id, $type, $scheme = array()) {
 /**
  * Register Color Schemes with Javascript
  */
-function tripald3_register_colorschemes() {
+function tripald3_register_colorschemes($default) {
 
   $jsSettings = array('tripalD3' => array());
 
   $schemes = tripald3_get_color_schemes();
   foreach ($schemes as $id => $scheme) {
-
-    $jsSettings['tripalD3']['colorSchemes'][$id]['name'] = $scheme['name'];
-    $jsSettings['tripalD3']['colorSchemes'][$id]['quantitative'] = tripald3_get_scheme_colors($id, 'quantitative', $scheme);
-    $jsSettings['tripalD3']['colorSchemes'][$id]['categorical'] = tripald3_get_scheme_colors($id, 'categorical', $scheme);
+    $jsSettings['colorSchemes'][$id]['name'] = $scheme['name'];
```

Comment:
Why did you drop the tripalD3 key here? This was a Drupal standard to reduce the chance of collisions in js settings between modules.